### PR TITLE
[update] mysql test correction

### PR DESCRIPTION
**Pull request by Stefan365**
Mysq test correction on jenkins server.
//test: @integrations-db-to-db-mysql

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java [hunk 1]
```diff
@@ -40,6 +40,7 @@ public static Connection getConnection(String dbType) {
                 break;
         }
 
+        log.info("mysql should be true:*{}*", connectionsInfoMap.containsKey(dbType));
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java]: assert instead of log?

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java [hunk 2]
```diff
@@ -40,6 +40,7 @@ public static Connection getConnection(String dbType) {
                 break;
         }
 
+        Assertions.assertThat(connectionsInfoMap.containsKey(dbType)).isTrue();
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java]: assertThat(connectionsInfoMap).containsKey(dbType); will be better

Comment (Stefan365) [reply utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java]: np

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**